### DISCUSSION TITLE
[Feature] Enhanced Memories (Spotlight on person)

### DISCUSSION THREAD

**Discussion by tomsalden** [immich-app/immich#8654] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Like already mentioned in #7437, the _App that shall not be named_ has multiple types of memories. In addition to a "trips" feature, I would like to suggest the "Spotlight on a person". I really enjoy this feature in the other App.

Of course, Immich already has a feature to group photos of persons, however this is different from the people page. What I would like to suggest is that these moments are a small (maybe varying) collection of highlights of a person.

When I look at current implementations, this feature seems to collect around 15 photos of a person to combine into a memory. A nice part of this feature is that the photos have a relatively even distributed over the years, if there are enough photos. This results in a nice flashback of this person over the years. 

The way to select them could be similar as DawidPietrykowski suggested for #7437, but then tailored to persons.

Possibly, these moments could appear inside the person's page, to prevent cluttering the memories with person spotlights. Additionally, the birthday feature could be used to show how old the person is in each photo in the memory.

Is this a quality-of-life addition that others are interested in as well?

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by djjudas21** [comment]
I like this idea. I would like to further suggest that a person spotlight appears on the day of that person's birthday. For example, yesterday was my brother's birthday and at the top of my iPhone gallery was a little slideshow of photos of him.

**Comment by gaurav-yadav** [comment]
https://github.com/immich-app/immich/pull/16764 made a draft PR we can discuss the approach and then i can build it.. open for inputs

**Comment by bo0tzz** [comment]
#2836
